reference/src/client/test/sa_key_unwrap_aes_gcm.cpp

```diff
@@ -129,7 +129,7 @@ namespace {
         ASSERT_NE(wrapping_key, nullptr);
 
         std::vector<uint8_t> aad = random(1024);
-        std::vector<uint8_t> tag = random(AES_BLOCK_SIZE);
```

Comment:
Update copyright year to include 2023. Please check other files to ensure copyright year is updated.

Reply:
Done